ReactAndroid/src/main/java/com/facebook/react/modules/statusbar/StatusBarModule.java

```diff
@@ -184,7 +185,28 @@ public void setStyle(@Nullable final String style) {
       return;
     }
 
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+      UiThreadUtil.runOnUiThread(
+        new Runnable() {
+          @TargetApi(Build.VERSION_CODES.R)
+          @Override
+          public void run() {
```

Comment:
You don't need to create another anonymous inner class. You shoudl be good to go with just a `if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {` inside the `run()` method.

Reply:
Done.